components/eventing-controller/cmd/eventing-controller/main.go

```diff
@@ -2,93 +2,94 @@ package main
 
 import (
 	"context"
-	"flag"
 	"os"
-	"time"
 
 	"k8s.io/apimachinery/pkg/runtime"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	ctrl "sigs.k8s.io/controller-runtime"
-	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 
+	"github.com/go-logr/zapr"
+
+	"github.com/kyma-project/kyma/components/eventing-controller/log"
+	"github.com/kyma-project/kyma/components/eventing-controller/options"
 	"github.com/kyma-project/kyma/components/eventing-controller/pkg/commander/beb"
 	"github.com/kyma-project/kyma/components/eventing-controller/pkg/commander/nats"
 	"github.com/kyma-project/kyma/components/eventing-controller/reconciler/backend"
 )
 
 func main() {
-	logger := ctrl.Log.WithName("setup")
-	scheme := runtime.NewScheme()
-	restCfg := ctrl.GetConfigOrDie()
+	setupLogger := ctrl.Log.WithName("setup")
+
+	opts := new(options.Options)
```

Comment:
While this is working too I would prefer a today more common `opts := options.New()`.